src/mongo/nodes.ts

```diff
@@ -268,7 +268,10 @@ export class MongoCollectionNode implements INode {
 	}
 
 	private find(args?: any): Thenable<string> {
-		const maxDocs = 20;
+		let maxDocs = <number>vscode.workspace.getConfiguration().get('cosmosDB.mongo.maxDocs');
```

Comment:
Is getConfiguration method guaranteed not to return null? Otherwise this might be a potential null ref error.